app/scripts/models/account.js

```diff
@@ -142,6 +142,10 @@ define([
         });
     },
 
+    isAuthenticated: function () {
```

Comment:
The name is a bit misleading since the session token can be present but expired. We should probably use `this._fxaClient.isSignedIn` to avoid those cases.